src/services/fileDestinationUrl.service.test.ts

```diff
@@ -1,20 +1,40 @@
+/* eslint-disable import/first --
+ * jest hoists mock() calls to the top and these are used in the mocks
+ */
+const sendMock = jest.fn();
+const getSignedUrlMock = jest.fn();
```

Comment:
It looks like we avoid doing this in the health check tests below; what's the difference?

Reply:
I think we should probably disable this for all tests, since this seems like an issue that will come up in other places too

Reply:
That's fine with me -- I'll include a comment that if / when we create better mock tooling we should re-enable the rule (since by disabling it we lose lint protections from imports being done in wrong places / mid function / etc)